test/elixir_sense/definition_test.exs

```diff
@@ -70,6 +70,17 @@ defmodule ElixirSense.Providers.DefinitionTest do
     assert read_line(file, {line, column}) =~ "function_arity_one"
   end
 
+  test "find definition of delegated functions" do
+    buffer = """
+    defmodule MyModule do
+      String.length("elixir")
```

Comment:
Is this the test you're intending? Shouldn't this reference `defdelegate` in some capacity, per the name?

Reply:
This test works because String.length/1 in the standard library is a `defdelegate`.  Without this PR "go to definition" for this function fails.  

Looking at this again I do agree testing in this way is brittle as the implementation details of the standard library can change at any time.  I will push commits modifying this this so it tests a `defdelegate` in the test buffer instead.